doc/SCALING.md

```diff
@@ -0,0 +1,106 @@
+
+And there are other factors that might boost your need for scaling:
+- smaller load on each instance, might result in better, more stable performance
+- for some reason an application instance dies, the less call will be affected in the ICE reconnect
+
+
+## Scaling with STUNner
+
+STUNner has a full support for horizontal scaling. That means the number of pods can be increased and decreased according to the user's needs. 
+In case the user wants to scale the instances of the `stunner` deployment, the Kubernetes built in `HorizontalPodAutoscaler` can be used. The triggering event can be based on different metrics but it is advised to use the currently utilized amount of CPU. If so, it is important to state that the HPA [uses the requested amount of resources](https://pauldally.medium.com/horizontalpodautoscaler-uses-request-not-limit-to-determine-when-to-scale-97643d808997) to determine when to scale-up or down the number of instances.
+
+###  Scaling-up (out)
+
+When adding new instances to the existing `stunner` replica set we don't have much to worry about. The infrastructure for existing calls won't get interrupted, it stays the same as before the upscale event. Only when new user calls come in the cloud loadbalancer have an additional endpoint to choose from. This way we can achieve that STUNner is never going to be the bottleneck in the system. Obviously, if you have the needed computational power under your cluster.
+
+Scaling-up the number of instances in the `stunner` deployment should be done based on the CPU usage of the replicas. As it was mentioned in the [section above](#scaling-with-stunner) the `HorizontalPodAutoscaler` is using the requested resources to determine when to scale up.
+
+### Scaling-down (in)
+
+When removing existing instances from the `stunner` replica set there are some things to keep in mind. What happens to the existing calls on a to-be-removed pod? Is there a way to keep them? 
+
+In worst case scenario all call connections will be dropped and the pod terminates itself, thus we lost all running connections/calls on the removed (scaled down) STUNner instance. A slightly better scenario is that the [ICE restart](https://developer.mozilla.org/en-US/docs/Web/API/RTCPeerConnection/restartIce) mechanism will kick at client side, basically it will reset the current ICE candidates and reconnect again the same way it did initially but it takes a second or two and not even supported by all browsers. 
+
+The third option is that STUNner [has full support](#more-on-the-graceful-shutdown) for graceful shutdown in Kubernetes. This means that `stunner` pods will remain alive as long as there are active allocations via the embedded TURN server, and a pod will automatically remove itself once all allocations through it are deleted or timed out. It is important that the *terminating* pod will not be counted in by the `HorizontalPodAutoscaler` as a running replica, thus its CPU usage won't be taken into account either. 
```

Comment:
I don't understand why this is an "option". Please set this as the only alternative, we support this by default. Why make people worry? Maybe after the "default" you can mention that one can switch graceful restarts off and then they will fall back to ICE restarts.